includes/admin_import.php

```diff
@@ -353,7 +354,11 @@ function tsml_import_page()
                                 <?php foreach ($tsml_data_sources as $feed => $properties) { ?>
                                     <tr data-source="<?php echo $feed ?>">
                                         <td class="small ">
-                                            <form method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
+                                            <?php
+                                            $refresh_form_name = 'frm_' . tsml_make_refresh_attr_name(@$properties['name'], 'frm');
+                                            ?>
+
+                                            <form name="<?php echo $refresh_form_name ?>" method="post" action="<?php echo $_SERVER['REQUEST_URI'] ?>">
```

Comment:
do we need this form name? what does it do?

Reply:
Part of the auto-refresh that we decided to wait on till this week.

Reply:
ah this is still work-in-progress? it would make sense to wait on this until the functionality is ready especially since we just rolled out such a big change